src/analyzer.ts

```diff
@@ -94,9 +98,7 @@ export class Analyzer {
    * Note: this feature is experimental.
    */
   _fork(): Analyzer {
-    const analyzer = new Analyzer({});
```

Comment:
FYI, the error was that `{}` didn't match options because it doesn't have `urlLoader`